File: app/src/main/java/com/gelostech/dankmemes/models/UserModel.java
package com.gelostech.dankmemes.models;

/**
 * Created by tirgei on 7/7/17.
 */

public class UserModel {
    private String userName, userId;

    public UserModel(){}

    public UserModel(String userName, String userId){
        this.userName = userName;

        this.userId = userId;
    }


    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }
}

File: app/src/main/java/com/gelostech/dankmemes/data/models/UserModel.kt
package com.gelostech.dankmemes.data.models

import java.io.Serializable

data class UserModel(
        var userId: String? = null,
        var userName: String? = null,
        var userToken: String? = null,
        var userAvatar: String? = null,
        var userEmail: String? = null,
        var dateCreated: String? = null,
        var userBio: String? = null
): Serializable